DuckDuckGo/Tab/Navigation/ExternalAppSchemeHandler.swift

```diff
@@ -78,6 +79,11 @@ extension ExternalAppSchemeHandler: NavigationResponder {
             }
         }
 
+        // proceed with web extension URLs
+        guard !externalUrl.isWebExtensionUrl else {
```

Comment:
maybe add the scheme to `URL.isExternalSchemeLink` in URLExtension to keep those in 1 place?